### PR TITLE
Update signer and core links

### PR DESCRIPTION
**Pull request by BowTiedDevOps**
Update signer version to 3.0.0.0.0.0 and node version to 3.0.0.0.0 in all places

### CODE REVIEW

File: guides-and-tutorials/running-a-signer/README.md [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
+
+```
+[miner]
+mining_key = "private key" ## may be the same as [node].seed
+```
+
+**Important note**:
+Any unrecognized and uncommented config value _will_ cause the binary to panic.
+
+The output will specify which configuration key caused the panic (there may be more than one, but the message will only panic on the first unrecognized key) in the format:
```
Comment (wileyj) [inline guides-and-tutorials/running-a-signer/README.md:27]:
```suggestion
The output will specify which configuration key caused the panic (there may be more than one, but the message will only show the first unrecognized key) in the format:
```

File: guides-and-tutorials/running-a-signer/README.md [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
 
-{% hint style="info" %}
 #### Current Signer and Stacks Node Versions
 
 For quick reference, here are the current latest versions you'll want to be running as a signer. If you don't yet have your signer up and running, this guide will walk you through that.
 
-* Signer - [2.5.0.0.5.2](https://hub.docker.com/layers/blockstack/stacks-signer/2.5.0.0.5.2/images/sha256-fc29a7c22f236f91270fb1aa58cfb4dd8dcd6b1daa0812e16df0bdc7643cb6ac?context=explore)
-* Stacks Node - [2.5.0.0.7](https://hub.docker.com/layers/blockstack/stacks-core/2.5.0.0.7/images/sha256-71d3eb305b5c1b68cd44904a7bcd6e5f92542135a7218762cdf27a46acaff69b?context=explore)
-{% endhint %}
+- [Binaries](https://github.com/stacks-network/stacks-core/releases)
```
Comment (wileyj) [inline guides-and-tutorials/running-a-signer/README.md:40]: maybe point to https://github.com/stacks-network/stacks-core/releases/latest  (which won't point to the signer binary specifically)?

Comment (kenrogers) [reply guides-and-tutorials/running-a-signer/README.md]: Done ✅

File: guides-and-tutorials/running-a-signer/README.md [hunk 3]
```diff
@@ -273,24 +300,24 @@ For the Stacks node to use this archive, you must specify `working_dir` in your
...
-* You will set working\_dir to equal ”/Users/blah”
-  * Note that the string does not include the “xenon” part
+- The snapshot is available at /Users/blah/xenon
+- You will set working_dir to equal ”/Users/blah”
+  - Note that the string does not include the “xenon” part
 
 #### Run a Stacks Node with Docker
 
-You can run the Stacks node as a Docker container using the `blockstack/stacks-core` image, currently on [version 2.5.0.0.7](https://hub.docker.com/layers/blockstack/stacks-core/2.5.0.0.7/images/sha256-71d3eb305b5c1b68cd44904a7bcd6e5f92542135a7218762cdf27a46acaff69b?context=explore). When running the Docker container, you’ll need to ensure a few things:
+You can run the Stacks node as a Docker container using the `blockstack/stacks-core` image, currently on [version 3.0.0.0.0.0](https://hub.docker.com/layers/blockstack/stacks-signer/3.0.0.0.0.0/images/sha256-5406912473befe62ce21f1630908b01960bae29b7bef306fd5669037ac8e112b?context=explore). When running the Docker container, you’ll need to ensure a few things:
```
Comment (wileyj) [inline guides-and-tutorials/running-a-signer/README.md:309]:
```suggestion
You can run the Stacks node as a Docker container using the `blockstack/stacks-core` image, currently on [version 3.0.0.0.0](https://hub.docker.com/layers/blockstack/stacks-core/3.0.0.0.0/images/sha256-7e0ce69dff7198ce0dd0f44676f065abd6f834ed6d9a396856a36fd422ba6134?context=repo). When running the Docker container, you’ll need to ensure a few things:
```

Comment (wileyj) [reply guides-and-tutorials/running-a-signer/README.md:309]: this was pointing to the signer image. assume we want stacks-core image here

**Review by wileyj — APPROVED** [review]

**Pull request merged**